wagon-providers/pom.xml

```diff
@@ -52,24 +51,11 @@ under the License.
         <artifactId>httpclient</artifactId>
         <version>4.5.14</version>
       </dependency>
-      <!-- Overwrite transitive dependency version of HttpClient -->
-      <dependency>
-        <groupId>commons-codec</groupId>
-        <artifactId>commons-codec</artifactId>
-        <version>1.16.1</version>
-        <scope>runtime</scope>
-      </dependency>
       <dependency>
         <groupId>org.apache.httpcomponents</groupId>
         <artifactId>httpcore</artifactId>
         <version>4.4.16</version>
       </dependency>
-      <dependency>
-        <groupId>org.apache.sshd</groupId>
-        <artifactId>sshd-core</artifactId>
-        <version>0.8.0</version>
-        <scope>test</scope>
-      </dependency>
```

Comment:
Not required?

Reply:
its required but only in one specific module here https://github.com/apache/maven-wagon/pull/135/files/72c1253cbac6240af6290ef01b40a8027a78f8a3#diff-15f3b17854f455ff3800ee2b45fa961a70bae1876256de66dc9bcd06bc19e1b3R58